src/patterns/send-a-web-form/index.njk

```diff
@@ -19,13 +19,11 @@ Do not use this pattern if the user needs to speak to someone at the contact cen
 
 ## How to use this pattern
 
-To help contact staff deal with web form enquiries, you should provide a set of categories based on the most common user needs from research.
+Give users a set of categories to choose from on the web form. These should be based on the most common user needs identified through user research. This helps contact centre staff deal with enquiries.
 
-You should only ask the user to provide the contact details the service needs to handle the request.
+Only ask the user for the contact details the service needs to handle the request.
 
-For Census 2021, a respondent could send a web form to the contact centre to receive the appropriate support to complete the Census.
-
-The census web form included the following, in this order:
+The web form pattern includes the following, in this order:
```

Comment:
I thought we were making it clear that these are just an example of what you could use in the web form pattern.  Otherwise, I think a user would read this and think they have to ask the user for e.g. what country they live in every time they want to create a web form.

Reply:
@jrbarnes9 can you discuss with Laura. I think we are on the mindset of removing a census relationship.

Reply:
Agree, that we should move away from census-based examples. Suggest changing this list to the following:

The following could be included on a web form:
What do you need help with? (radio selection)
Please provide more detail (field)
Name (field)
Email address (field)
Send message (button)

Reply:
Let me get your suggestion added. Thanks.

Reply:
Content changes have been added as per the documentation. Census theme removed in favour of our default ONS look and feel.